operations/mimir-mixin/dashboards/dashboard-queries.libsonnet

```diff
@@ -233,5 +234,30 @@ local utils = import 'mixin-utils/utils.libsonnet';
         sum(rate(thanos_objstore_bucket_operations_total{%(namespaceMatcher)s}[$__rate_interval]))
       ||| % variables,
     },
+
+    ingester: {
+      ingestOrClassicDeduplicatedQuery(perIngesterQuery, groupByLabels=''):: |||
+        # Classic storage
+        sum by (%(groupByCluster)s, %(groupByLabels)s) (%(perIngesterQuery)s)
+        / on (%(groupByCluster)s) group_left()
+        max by (%(groupByCluster)s) (cortex_distributor_replication_factor{%(distributor)s})
+        or on (%(groupByCluster)s)
```

Comment:
I don't understand one thing: what is this `on (%(groupByCluster)s)` (in this line) used for?